.github/workflows/prod_deploy.yaml

```diff
@@ -7,18 +7,27 @@ jobs:
   invoke-prod-deploy:
     runs-on: ubuntu-latest
     steps:
-      - name: deploy
+      - name: deploy_gcp
         run: |
           DEPLOY_BATCH_URL=$(curl --fail --silent --show-error -X POST \
               -H "Authorization: Bearer ${{ secrets.CI_TOKEN }}" \
               -H "Content-Type:application/json" \
               -d '{"steps": ["deploy_auth", "deploy_batch", "deploy_ci", "deploy_hailgenetics_image", "deploy_memory", "deploy_wheel", "upload_query_jar"], "sha": "${{ github.sha }}"}' \
               https://ci.hail.populationgenomics.org.au/api/v1alpha/prod_deploy)
-          echo DEPLOY_BATCH_URL=$DEPLOY_BATCH_URL >> $GITHUB_ENV
+          echo DEPLOY_BATCH_URL_GCP=$DEPLOY_BATCH_URL >> $GITHUB_ENV
+
+      - name: deploy_azure
+        run: |
+          DEPLOY_BATCH_URL=$(curl --fail --silent --show-error -X POST \
+              -H "Authorization: Bearer ${{ secrets.CI_TOKEN_AZURE }}" \
+              -H "Content-Type:application/json" \
+              -d '{"steps": ["deploy_auth", "deploy_batch", "deploy_ci"], "sha": "${{ github.sha }}"}' \
```

Comment:
Why only those few steps here? We'll definitely need the Query JAR, for example, to run Query on Batch.

Can we share the same templated invocation and just interpolate the token + URL, so we don't run the risk of getting out-of-sync between GCP and Azure?

Reply:
Currently on Azure we have only the auth, batch and ci deployed. Running the ci deploy command  with all of them failed on `deploy_hailgenetics_image`. To be fair I hadn't checked the others but I haven't even tested them yet.

The current azure instructions stop at `deploy_batch`. I don't even know if the rest are implemented yet or not but I can ask.

Reply:
Ok, I've confirmed that `deploy_hailgenetics_image` and `deploy_wheel` are not implemented. The others I've successfully run and added to the ci hook as well.

Reply:
Thanks!